src/main/java/org/isf/telemetry/gui/TelemetryEdit.java

```diff
@@ -337,50 +337,34 @@ private Map<String, Boolean> buildConsentData(List<CheckBoxWrapper> permissions)
 	}
 
 	private ActionListener buildAskMeLaterButtonActionListener(TelemetryManager telemetryManager) {
-		return new ActionListener() {
-
-			public void actionPerformed(ActionEvent e) {
-				dispose();
-			}
-		};
-
+		return e -> dispose();
```

Comment:
true, and if I recall well, we decided also to use extended name, like `actionEvent` in this case?

```suggestion
		return actionEvent -> dispose();
```

Reply:
@mwithi Done and pushed